st2actions/st2actions/query/mistral/v2.py

```diff
@@ -42,13 +42,25 @@ def query(self, execution_id, query_context):
             status, output = self._get_workflow_result(exec_id)
             if output and 'tasks' in output:
                 LOG.warn('Key conflict with tasks in the workflow output.')
+        except requests.exceptions.ConnectionError:
+            msg = 'Unable to connect to mistral.'
+            LOG.exception(msg)
+            return (LIVEACTION_STATUS_RUNNING, {'error': msg})
```

Comment:
We update the whole results objects every time `query` method returns, right?

If so, this means that during failure we will store error in the `result` attribute of a live action, right? Is this desirable?